roles/pulp3-resource-manager/templates/pulp_resource_manager.service.j2

```diff
@@ -1,3 +1,23 @@
+{% if pulp_container_deployment %}
+[Unit]
+Description=Pulp Resource Manager
+Wants=syslog.service postgresql.service redis.service
```

Comment:
Should this also want `network-online.target`? Could this be shared between the container and non-container definitions?